src/main/java/duke/Event.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+
+/**
+ * Encapsulates the Event class which is a sub-class of the Task class.
+ */
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
+
+    /**
+     * Initialises the from and to field in LocalDate type.
```

Comment:
@params and @throws should have more details as well to meet coding standards.
E.g. @param description Description of task